scipy/linalg/_solvers.py

```diff
@@ -150,7 +150,7 @@ def _solve_discrete_lyapunov_bilinear(a, q):
 
 def solve_discrete_lyapunov(a, q, method=None):
     """
-    Solves the discrete Lyapunov equation :math:`A'XA-X=-Q`.
+    Solves the discrete Lyapunov equation :math:`AXA^H - X = -Q`.
```

Comment:
It seems to me the algorithms used here don't need to assume anything about `Q`. AFAICS, whether they work depends only on spectrum of `A`.